var/spack/repos/builtin/packages/perl-class-tiny/package.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+# Copyright 2023 EMBL-European Bioinformatics Institute
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
```

Comment:
This seems to be unrelated to this PR, and a duplicate of #41513

Reply:
Indeed, sorry about that.